.idea/compiler.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="1.8" />
+    <bytecodeTargetLevel target="1.8">
```

Comment:
The whole .idea folder should be in your gitignore. 
Nothing in this folder should be checked into git.